### PR TITLE
Wrapper consistency (resolves #86)

### LINKED ISSUE #86: Increase consistency among shell wrappers

### PR DESCRIPTION
**Pull request by hannes-ucsc**
Resolves #86 by 
- adding the permission fix to all wrappers, except conductor
- have the permission fix use the trap shell built-in
- use `env` to locate bash in the wrapper shebangs
- use `set -e` in all wrappers

### CODE REVIEW

**Comment by hannes-ucsc** [comment]
@fnothaft, could you review this?

**Comment by jvivian** [comment]
LG2M

File: conductor/runtime/wrapper.sh [hunk 1]
```diff
@@ -1,9 +1,17 @@
-#! /bin/bash
+#!/usr/bin/env bash
+set -e
+
+# Fix ownership of output files
```
Comment (fnothaft) [inline conductor/runtime/wrapper.sh]: This image does _not_ write to /data.

**Comment by fnothaft** [comment]
LGTM, except we should remove the fix from the conductor image as it should always write to HDFS remotely.

**Comment by hannes-ucsc** [comment]
Removed the permission fix from conductor.

**Pull request merged**